frontend/assets/javascripts/src/modules/form/helper/formUtil.js

```diff
@@ -0,0 +1,64 @@
+define([
+    'src/utils/helper'
+], function (utilsHelper) {
+
+    /**
+     * Filter only the elements that require validation, 'input', 'textarea', 'select' input elements that are
+     * required or have a validation profile
+     * @param elems
+     * @returns {*}
+     */
+    var inputsToValidate = function (elems) {
+        return elems.filter(function (elem) {
+            var nodeName = elem.nodeName.toLowerCase();
+            var inputTypes = ['input', 'textarea', 'select'];
+            var required = elem.hasAttribute('required');
+            var validationProfile = elem.getAttribute('data-validation');
+
+            if ((inputTypes.indexOf(nodeName) !== -1) && (required || validationProfile)) {
+                return true;
+            }
+        });
```

Comment:
This function seems to return `true` or `undefined`, ideally there should be one return type.